### DISCUSSION TITLE
Why can't I set eastasia.family using font()?

### DISCUSSION THREAD

**Discussion by RyEfreeti** [davidgohel/flextable#604] [Q&A]
I want to set different fonts for Chinese and English at the same time, but eastasia.family and hansi.family don't work all fonts follow the fontname setting by default

ft <- font(ft,fontname ="Times New Roman",esatia.family="宋体",part="all")

**Comment by davidgohel** [comment]
can you provide a reproducible example? It's hard to help without a code that show the issue you met

**Reply by RyEfreeti** [reply]
Sorry，I apologize for not getting back to you until now.

Here is the sample code


```
library(flextable)
library(dplyr)
library(officer)

set_font <- function(df, c_bold){
  
  tbl <- flextable(df)
  tbl <- tbl %>%
    autofit() %>%
    bold(j = 1, i = c_bold) %>%
    bold(part = "header") %>%
    align(align = "left", part = "all") %>%
    font(fontname = "宋体", part = "all", hansi.family = "Times New Roman", eastasia.family = "宋体") %>%
    fontsize(size = 9, part = "all") %>%
    padding(padding = 2, part = "all")
  
  return(tbl)
}

df_example <- data.frame(
  Name = c("中文", "Bob", "Charlie"),
  Score = c(85, 92, 88)
)

tbl <- set_font(df_example, c_bold = 1)

doc <- read_docx()

doc <- doc %>%
  body_add_flextable(tbl)

print(doc, target = "example.docx")
```

The font in the generated word is all "宋体"， But I'd like it to be "宋体" in Chinese and "Times New Roman" in English.
[example.docx](https://github.com/davidgohel/flextable/files/14392957/example.docx)
